src/main/java/me/dinowernli/grpc/polyglot/testing/TestUtils.java

```diff
@@ -31,11 +31,11 @@ public static Path getWorkspaceRoot() {
 
   public static ImmutableList<String> makePolyglotCallArgs(String endpoint, String method) {
     return ImmutableList.<String>builder()
-        .add(makeArgument("command", "call"))
-        .add(makeArgument("endpoint", endpoint))
         .add(makeArgument("proto_discovery_root", TestUtils.TESTING_PROTO_ROOT.toString()))
-        .add(makeArgument("full_method", method))
         .add(makeArgument("add_protoc_includes", TestUtils.getWorkspaceRoot().toString()))
+        .add("call")
+        .add(makeArgument("endpoint", endpoint))
```

Comment:
Does this mean that the arguments now need to be passed in a specific order?

Reply:
Yes that's true. The command based approach takes this form:
`polyglot [global options] [command] [command options]`

I separated the options into global ones (applicable for every command), and command specific ones. I inferred this knowledge from the service calls in Main. This is also why I had to change the ordering in a couple of places (tests).

But of course this also means a bigger change to the original usage and hence impacting users, possibly breaking scripts, however I still think it's cleaner and the bump in Major version signals this change.

Ultimately it's up to you and what you feel comfortable with. I could also go back to an options-only approach where we specify the command as an option (like the original code).

Reply:
Git does it the same way for example:
`git -C <path> commit --amend -m 'Foo'`

`-C` is a global option, `commit` is a command, `--amend` and `-m` are command specific options. These need to be in the right order as well.

Reply:
Yup, makes sense. I think that's worth mentioning in the release notes.